interface/interface.dm

```diff
@@ -124,7 +124,9 @@ Hotkey-Mode: (hotkey-mode must be on)
 \tm = me
 \tt = say
 \to = OOC
+\tl = LOOC
```

Comment:
uhh?

Reply:
that's from another pr I made that got merged that I forgot to put in there, sorry, I should include that in the description

Reply:
L ISNT the looc hotkey though. CTRL+O is.

Reply:
nvm yes it is but only outside of hotkey mode. It's still CTRL+O in hotkey mode.

Reply:
actually, is the old_hotkeys mode even used at all?

Reply:
I have
no fucking clue
but yes this should be in here

Reply:
hotkeys mode must be off for this to work. This keybinding doesn't belong to this list.